src/components/general-table/GeneralTable.js

```diff
@@ -47,9 +47,9 @@ const filterParams = (chipsArray) => {
 };
 
 const GeneralTable = ({
-  historyProp,
-  locationProp,
-  navigateProp,
+  // historyProp,
```

Comment:
Those properties cannot be removed once we are using on inventory side.
When and IF the changes we made on inventory side, we remove.
The context and props can coexist.
Thoughts?

Reply:
Context and Props override each other because both of them modifying the general table